src/vegas_mode/plugin_adapter.py

```
@@ -0,0 +1,316 @@
+
+            if valid_images:
+                logger.debug(
+                    "Got %d native images from %s (total width: %dpx)",
+                    len(valid_images), plugin_id,
+                    sum(img.width for img in valid_images)
+                )
+                return valid_images
+
+            return None
+
+        except (AttributeError, TypeError, ValueError, OSError) as e:
```

Ruff (0.14.14)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)